x-pack/plugins/ml/public/embeddables/anomaly_charts/anomaly_charts_initializer.tsx

```diff
@@ -136,7 +171,7 @@ export const AnomalyChartsInitializer: FC<AnomalyChartsInitializerProps> = ({
             defaultMessage="Confirm configurations"
```

Comment:
The 'Confirm' button should be disables if no jobs / groups are selected:

<img width="346" alt="Screenshot 2024-05-20 at 10 24 07" src="https://github.com/elastic/kibana/assets/7405507/394a47bd-b624-4f13-997a-b2714a114845">

Reply:
Updated here https://github.com/elastic/kibana/pull/183456/commits/8ac6e9c6102dd899014da6101ca8e2c709beec43